OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -1,4 +1,4 @@
-#region Copyright & License Information
+﻿#region Copyright & License Information
```

Comment:
Can you avoid committing these changes in general especially if it is the only change in the whole file?